### DISCUSSION TITLE
Position / coordinate types documentation

### DISCUSSION THREAD

**Discussion by conor909** [rnmapbox/maps#3156] [Ideas]
The `Position` type, is currently `[number, number] | number[]` but is it [ lat, long ] or [ long, lat ]?  I can see the prop `centerCoordinate` for `Camera` uses [ long, lat ].  Is the same everywhere?  I'm coming from gmaps where its usually [ lat, long ] so it would help if it was documented

**Comment by mfazekas** [comment]
It's long, lat. Same as used in GeoJSON spec.

https://stevage.github.io/geojson-spec/#section-3.1.1

Feel free to add PR to document that.
